src/Nethermind/Nethermind.Evm.Test/TransactionProcessorTests.cs

```diff
@@ -293,7 +293,9 @@ public void Balance_is_not_changed_on_call_and_restore()
             _stateProvider.GetBalance(TestItem.PrivateKeyA.Address).Should().Be(1.Ether());
         }
         
-        [Test]
+        //Inappropriate test
+        
+        /*[Test]
```

Comment:
Why is it commented?

Reply:
Did you give up the idea of verification of account cleanup, which we discussed?